### ISSUE TITLE
Allow images

### ISSUE THREAD

**Issue by jaredpalmer** [algolia/algolia-sitemap#107]
Google recognizes images thru sitemaps. You can add them by using this format: 

```xml
	<url>
		<loc>https://example.com/info/chapters</loc>
		<lastmod>2017-11-04T21:57:13+00:00</lastmod>
		<image:image>
			<image:loc>https://example.com/wp-content/uploads/2016/06/myimage.jpg</image:loc>
			<image:title><![CDATA[myimage title]]></image:title>
		</image:image>
		<image:image>
			<image:loc>https://example.com/info/wp-content/uploads/2016/07/chapter-map.jpg</image:loc>
			<image:caption><![CDATA[Chapter Map]]></image:caption>
		</image:image>
	</url>
```

It would be nice to see this supported by letting lib consumers pass an array of images:

```js
function hitToParams(hit) {
  const url = ({ lang, hit }) => `https://example.com/img/${hit.objectID}`;
  const loc = url({ lang: 'en', hit });
  const lastmod = new Date().toISOString();
  const priority = Math.random();
  return {
    loc,
    lastmod,
    priority,
    images: [{
      loc: process.env.REACT_APP_URL + '/' + hit.imagePath,
      title: `<![CDATA[${hit.title}]]>`,
    }],
  };
}
```

Thoughts?

**Comment by Haroenv** [comment]
released as 2.1.0

**Comment by jaredpalmer** [comment]
Holla
